src/modelgauge/annotation_pipeline.py

```diff
@@ -114,6 +114,7 @@ def __init__(self, annotators: dict[str, Annotator], workers=None, cache_path=No
             workers = 8
         super().__init__(thread_count=workers, cache_path=cache_path)
         self.annotators = annotators
+        self.annotation_counts = {uid: 0 for uid in annotators}
```

Comment:
This is totally fine here, but for similar cases in the future, check out defaultdict.